app/controllers/api/v1/claims_controller.rb

```diff
@@ -0,0 +1,10 @@
+module API
+  module V1
+    class ClaimsController < ActionController::Base
+      def show
+        @users = User.claiming_today
```

Comment:
Even though it can be muscle memory inside a controller, you don't need an instance variable here:

```ruby
def show
  render json: User.claiming_today
end
```